dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1339,17 +1340,41 @@ class WebCompileTest {
     });
   }
 
-  /// Obtains the size and gzipped size of a file given by [fileName].
-  static Future<Map<String, int>> getSize(String fileName, {required String metric}) async {
+  /// Obtains the size and gzipped size of both [dartBundleFile] and [buildDir].
+  static Future<Map<String, int>> getSize(
+    String buildDir,
+    String dartBundleFile, {
```

Comment:
If it's not too much work, could you change this to take a list of folders and files instead? We probably are going to want to track other files in the future.

Reply:
SGTM

Reply:
done